frontend/src/individual_edit.tsx

```diff
@@ -603,50 +621,36 @@ export function IndividualEdit({ id }: { id: string | undefined }) {
                     }}
                   />
                 </div>
-                <div className={style.flexRow}>
-                  <Autocomplete
-                    options={motherOptions ?? []}
-                    value={
-                      motherOptions.find(
-                        (option) => option.value == individual?.mother?.number
-                      ) ?? motherOptions[0]
-                    }
-                    getOptionLabel={(option: OptionType) => option.label}
-                    renderInput={(params) => (
-                      <TextField
-                        {...params}
-                        label="Mor"
-                        className={style.control}
-                        variant={inputVariant}
-                        margin="normal"
-                      />
-                    )}
-                    onChange={(event: any, newValue: OptionType | null) => {
-                      updateField("mother", asIndividual(newValue?.value));
-                    }}
-                  />
-                  <Autocomplete
-                    options={fatherOptions ?? []}
-                    value={
-                      fatherOptions.find(
-                        (option) => option.value == individual?.father?.number
-                      ) ?? fatherOptions[0]
-                    }
-                    getOptionLabel={(option: OptionType) => option.label}
-                    renderInput={(params) => (
-                      <TextField
-                        {...params}
-                        label="Far"
-                        className={style.control}
-                        variant={inputVariant}
-                        margin="normal"
-                      />
-                    )}
-                    onChange={(event: any, newValue: OptionType | null) => {
-                      updateField("father", asIndividual(newValue?.value));
-                    }}
-                  />
-                </div>
+                <Autocomplete
+                  options={breedingEvents ?? []}
+                  onChange={(event: any, newValue: Breeding | null) => {
+                    newValue && updateField("breeding", newValue?.id ?? null);
+                  }}
+                  defaultValue={individual.breeding}
+                  value={breedingEvents.find(
+                    (option: Breeding) =>
+                      option.id?.toString() == individual.breeding.toString()
+                  )}
+                  getOptionLabel={(option: Breeding) =>
+                    option.father && option.mother
+                      ? "[" +
+                        option.breed_date +
+                        "] " +
+                        option.father +
+                        " - " +
+                        option.mother
+                      : breedingLabel(individual)
+                  }
```

Comment:
Many of the old breeding events in the data don't have a breeding date but only a birth date. In those cases, you get something like 
```
[null] G123-1234 - G567-5678
```
To prevent this from happening we could take the birth date if there is no breeding date. And specify for the user which one we are showing. 
```suggestion
                  getOptionLabel={(option: Breeding) =>
                    option.father && option.mother
                      ? "[" +
                        (option.breed_date
                          ? "Parning " + option.breed_date
                          : "Födsel " + option.birth_date) +
                        "] " +
                        option.father +
                        " - " +
                        option.mother
                      : breedingLabel(individual)
                  }
```